src/pages/FindWorkflow/FindWorkflowModal.js

```diff
@@ -67,19 +70,25 @@ const suggestedWorkflowsList = [
 const FindWorkflowModal = ({ onDismiss }) => {
   const [selectedSubHeader, setSelectedSubHeader] = useState('browse-suggested-workflows')
   const [loading, setLoading] = useState()
+  const [workflowName, setWorkflowName] = useState('')
+  const [versionName, setVersionName] = useState('')
```

Comment:
small nit: since all these are method related I am thinking it might be better to name them as `methodName` and `methodVersionName`. What do you think?